models/metrics/metrics.go

```diff
@@ -43,12 +43,12 @@ func metricName(name string) string {
 // Init initializes the arn sdk model metrics. This should only be called by the tattler constructor or tests.
 func Init(meter metric.Meter) error {
 	var err error
-	events.sent, err = meter.Int64Counter(metricName("event_sent_total"), metric.WithDescription("total number of events sent by the ARN client"))
+	events.sent, err = meter.Int64Counter(metricName("event_sent"), metric.WithDescription("total number of events sent by the ARN client"))
```

Comment:
If we feel strongly about these metric name changes then we need to also update the vma scrape 

https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-operator?path=/manifests/configs/teams/runtime/tattler.k

that leads into AKSCCPMetrics and currently flowing

Reply:
in AKSCCPMetrics I'm currently see ArnSdkEventSentTotal logged. I don't see TotalTotal logged.

I also see "ArnSdkEventSentBytesTotal" and "ArnSdkPromiseTotal"